src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -64,13 +64,13 @@ function AnnotationBody({ annotation, settings }) {
             markdown={text}
             textClass={{
               AnnotationBody__text: true,
-              'is-hidden': isHidden(annotation),
+              'p-redacted-content': isHidden(annotation),
             }}
           />
         </Excerpt>
       )}
       {isCollapsible && (
-        <div className="AnnotationBody__collapse-toggle">
+        <div className="hyp-u-layout-row--justify-right">
```

Comment:
I noticed that the `hyp-u-layout-*` styles are currently not documented in the Layout section of the pattern library so I had to hunt around in the `frontend-shared` source to find the definitions of these styles and then lookup the details of the `flex` mixin. This wasn't difficult, but it did mean some indirection for what boils down to `display: flex; flex-direction: row; justify-content: flex-end;`

I'm not requesting a change here, just airing a general thought about discoverability of these utility styles going forwards.

Reply:
No, no, this is an _entirely reasonable_ point to make. You are right.